cas/remote.go

```diff
@@ -93,13 +153,40 @@ func (r Remote) Download(ds Store) (*Remote, error) {
 		return nil, fmt.Errorf("bad HTTP status code: %d", res.StatusCode)
 	}
 
-	key, err := ds.WriteACI(reader)
+	aciTempFile, err := ioutil.TempFile("", "")
```

Comment:
I am wondering whether we should try create this on the same filesystem as the rkt directory (e.g. as WriteACI does)